docs/source/working.rst

```diff
@@ -0,0 +1,530 @@
+
+.. note::
+   Our current recommended practice is to avoid periodicity and simplify the model such that the equilibrium is constant (e.g., removing tilt and gravity). The MBC is still required but it is not required to use different linearization at different azimuthal positions.
+
+One of the output of the linearization is the state matrix (`A`) which relates the system states to their time derivatives. An eigenvalue analysis of the state matrix provide the frequencies and damping of the system. 
+
+.. note::
+    Unlike a linear finite-element software, OpenFAST does not have a notion of stiffness and mass matrix. The underlying system of equation is non-linear, the frequencies of the system will vary with the operating conditions (e.g. wind speed, rotational speed).
+
+The sections below detail the process of obtaining a linear model with OpenFAST, and will focus on its application to obtain the frequencies and damping of the system modes.
+
+
+
+
+Linearized models for one simulation (manually)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+This section describes the key steps to generate a linearized model of the system with OpenFAST. 
+
+The steps to perform simple linearization analysis are given below:
+
+1. Edit the main `.fst` file, set `Linearize=True`
+
+2. Set the output format `OutFmt` to "ES20.11E3". The output files will be written with this high resolution, which is required for accurate eigenvalue analyses.
+
+.. warning::
+    Because the linearization output files are in ASCII format, the results of the eigenvalue analyses will be sensitive to the output resolution (`OutFmt`). It is therefore important to set this parameter with a large precision as mentioned above.
+
+3. There are two main methods to determine at which times the linearization will be made: 
+ 
+   - using `CalcSteady=False`, the user prescribes the times where linearization is to occur using `NLinTimes` and `LinTimes` 
+     (it is the responsibility of the user to provide times where the system is in equilibrium or at a periodic steady state, i.e. sufficiently long time); 
+     - `CalcSteady=True` (recommended approach), OpenFAST will automatically start the linearization when the system is at a periodic steady state (based on given tolerance `TrimTol`) and will perform `NLinTimes` linearizations over a rotor revolution. When a controller is used the option `CalcSteady` will also adjust the controller inputs (either Pitch, Yaw, or Generator Torque, based on the input `TrimCase`) such as to reach the rotational speed indicated by the initial condition. The `TrimGain` and `TrimTol` might need to be adjusted. 
```

Comment:
Suggest clarifying that Twr_Kdmp and Bld_Kdmp can be used to add damping to the tower and blades during the steady-state calculation.  These may be helpful to speed up the steady-state calculation and may be necessary if the tower and/or blades are otherwise unstable.  Once the steady-state solution is found.  Twr_Kdmp and Bld_Kdmp will not impact the linearization results (i..e., the linear solution will not have extra tower and blade damping).